sorc/CMakeLists.txt

```diff
@@ -1,13 +1,24 @@
 include(ExternalProject)
 
+##### CMAKE Arguments depending on exec_bin #########################
+
+list(APPEND COMP_ARGS "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}")
+if(${exec_bin} STREQUAL exec)
```

Comment:
What is the point of supporting multiple names for the executable install directory?

Reply:
@christopherwharrop-noaa, the UFS SRW App will be used as an operational model. We should change the app to meet the NCO standards for this. If you and other reviewers agree, we can use only one name as 'exec'. But I don't think this is what you want.